src/components/Form.js

```diff
@@ -20,24 +20,31 @@ const Form = () => {
 
     const bookObj = {
       ...book,
+      category: 'Action',
       item_id: Date.now().toString(),
     };
 
     dispatch(addBookAction(bookObj));
 
-    // clear values
     setBook({
       title: '',
       author: '',
     });
   };
 
   return (
-    <form method="POST" action="/" className="add">
+    <form method="POST" action="/" className="add" onSubmit={(e) => submitForm(e)}>
       <h3>ADD NEW BOOK</h3>
-      <input type="text" name="title" id="title" value={book.title} placeholder="Book title" onChange={handleChange} required />
-      <input type="text" name="author" id="author" value={book.author} placeholder="Author" onChange={handleChange} required />
-      <button type="submit" onClick={(e) => submitForm(e)}>Add Book</button>
+      <fieldset>
+        <input type="text" name="title" id="title" value={book.title} placeholder="Book title" onChange={handleChange} required />
+        <input type="text" name="author" id="author" value={book.author} placeholder="Author" onChange={handleChange} required />
+        <select id="author">
+          <option value="Action">Adventure</option>
+          <option value="Action">Thriller</option>
+          <option value="Action">Action</option>
+        </select>
```

Comment:
- [ ] (Optional) It would be nice to add an `eventlistener` to the `select` form control and then implement the React hook `useState` in such a way that the `option` selected becomes the current state and it is displayed. Currently, when I add a book, I still get action as the default `category`. You can read this [article](https://reactgo.com/react-hooks-forms/) as a guide. 👍